Content.Server/_NF/Trade/TradeCrateDestinationComponent.cs

```diff
@@ -0,0 +1,14 @@
+using Content.Shared._NF.Trade;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Server._NF.Trade;
+
+/// <summary>
+/// This is used to mark an entity to be used in a trade crates
+/// </summary>
+[RegisterComponent]
+public sealed partial class TradeCrateDestinationComponent : Component
+{
+    [DataField(customTypeSerializer: typeof(PrototypeIdSerializer<TradeCrateDestinationPrototype>))]
+    public string DestinationProto;
```

Comment:
protoid

Reply:
I had it as a ProtoId before, but it wouldn't work, complained about packet serialization.  If you know how to fix that, great, otherwise, this.  It should be a shared prototype, unsure why it wasn't working,

Reply:
yes you need to use `destComp.DestinationProto.Id` instead of `destComp.DestinationProto` in the OnTradeCrateInit method